Neos.Fusion/Classes/FusionObjects/AbstractArrayFusionObject.php

```diff
@@ -82,4 +86,86 @@ public function offsetUnset($offset)
     {
         unset($this->properties[$offset]);
     }
+
+    /**
+     * @param string|null $defaultFusionPrototypeName
+     * @return array
+     * @throws FusionException
+     * @throws \Neos\Flow\Configuration\Exception\InvalidConfigurationException
+     * @throws \Neos\Flow\Mvc\Exception\StopActionException
+     * @throws \Neos\Flow\Security\Exception
+     */
+    protected function evaluateNestedProperties(?string $defaultFusionPrototypeName = null): array
+    {
+        $sortedChildFusionKeys = $this->sortNestedProperties();
+
+        if (count($sortedChildFusionKeys) === 0) {
+            return [];
+        }
+
+        $result = [];
+        foreach ($sortedChildFusionKeys as $key) {
+            $propertyPath = $key;
+            if ($this->isUntyped($key) && $defaultFusionPrototypeName) {
```

Comment:
```suggestion
            if ($defaultFusionPrototypeName !== null && $this->isUntyped($key)) {
```

wenn $defaultFusionPrototypeName === null müsste das sorum minimal schneller sein